src/components/MdDrawer/MdDrawer.vue

```diff
@@ -11,8 +11,11 @@
   import MdOverlay from 'components/MdOverlay/MdOverlay'
   import MdPropValidator from 'core/utils/MdPropValidator'
 
+  import MdSwipeable from 'core/mixins/MdSwipeable/MdSwipeable'
+
   export default new MdComponent({
     name: 'MdDrawer',
+    mixins: [MdSwipeable],
```

Comment:
For the drawer element, we should use the swipeElement on drawers. It can't be window, because we need to ensure those right drawers will work.